### PR TITLE
Update EFS supported regions

### PR DESCRIPTION
**Pull request by juan131**
### *Issue #, if available:*

- fix https://github.com/aws-quickstart/quickstart-bitnami-wordpress/issues/23

### *Description of changes:*

This PR updates the EFS supported regions so the assertions admit London (eu-west-2) as a supported region.

As reported by @TrentDSD, EFS is explicitly supported there in that region.

### CODE REVIEW

**Comment by TrentDSD** [comment]
@agamzn @juan131 thanks for looking at this. Would I also need to point to a valid "BITNAMIWP" AMI for eu-west-2 in the /templates/webserver.template (line 364) file?

**Comment by agamzn** [comment]
@TrentDSD  - good catch! Yes. @juan131  - can you or @beltran-rubo add the AMI ID for eu-west-2 ?

**Comment by TrentDSD** [comment]
@agamzn @juan131 really grateful for this - I'm eager to get this deploying successfully to London and would rather do that direct from the Quickstart repo. Any chance of a new PR for the changes to /templates/webserver.template and /ci/taskcat.yml files? I don't know what's entailed in the creation of an AMI ID for "BITNAMIWP".

**Comment by agamzn** [comment]
@TrentDSD  - We keep track of the latest AMI IDs in each region and update the templates periodically. It saves a lot of work all around. We pass in specific filters to 'aws ec2 describe-images' to fetch those AMI IDs. Right now this feature is internal to our team (AWS QuickStart), but we're working to expose it. 

Based on the filters that we have right now, there's no AMI in eu-west-2. I'm reaching out to Bitnami via email to maintain momentum.

**Comment by andresbono** [comment]
Hi @TrentDSD, @agamzn I have just updated the AMI mapping manually. It includes the new AMI for the `eu-west-2` region:

https://github.com/tompizmor/quickstart-bitnami-wordpress/blob/e63f6ce144623581836fd36b7ff2ebca0f81aec1/templates/webserver.template#L365-L367
```
            "eu-west-2": {
                "BITNAMIWP": "ami-0a3572d36ec8b2f2f"
            },
```

The `ci/taskcat.yml` file is also updated.

File: templates/webserver.template [hunk 1]
```diff
@@ -342,37 +342,40 @@
     "Mappings": {
         "AWSAMIRegionMap": {
             "AMI": {
-                "BITNAMIWP": "bitnami-wordpress-5.0.2-0-r01-linux-debian-9-x86_64-high-availability-nami-hvm-ebs"
+                "BITNAMIWP": "bitnami-wordpress-5.0.3-5-linux-debian-9-x86_64-hvm-ebs-high-availability-nami"
```
Comment (andresbono) [inline templates/webserver.template]: @agamzn, I just noticed that we have changed the AMI name structure of our released images. It probably affects to the automated track of new WordPress AMIs.

I'll check the rationale of the change early next week and I'll get back to you.

Comment (TrentDSD) [reply templates/webserver.template]: **Huge** thanks for this @andresbono and @agamzn .

**Comment by agamzn** [comment]
The blockers for this PR are resolved - merging.

**Comment by juan131** [comment]
That's great!! Thanks @agamzn @andresbono

**Pull request merged**